master/internal/api_experiment.go

```diff
@@ -92,6 +92,33 @@ func (a *apiServer) GetExperiments(
 		if len(req.Users) != 0 && !found {
 			return false
 		}
+
+		if len(req.Labels) != 0 {
+			found = true
+
+			if len(v.Labels) == 0 {
+				return false
+			}
```

Comment:
nonblocking: It seems like there ought to be a way to get rid of this section, right?  Like shouldn't the `len(v.Labels) == 0` case be handled passively by whatever for loops are enforcing the logical AND?

Reply:
yeah that bothers me too but I couldn't immediately see a way of getting rid of that without making it less readable. :thinking: